src/core/Akka/Util/Internal/Extensions.cs

```diff
@@ -199,6 +200,12 @@ public static void ForEach<T>(this IEnumerable<T> source, Action<T> action)
                 action(item);
         }
 
+        /// <inheritdoc cref="ForEach{T}(IEnumerable{T}, Action{T})"/>
+        public static async Task ForEachAsync<T>(this IEnumerable<T> source, Func<T, Task> action)
```

Comment:
Instead of making a new ForEachAsync like this and have the CI/CD run everything because you changed the Akka project, why not use a simple `foreach()`?

Reply:
Ok, will do that!